<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderManager</class>
 <widget class="QWidget" name="OrderManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_manu">
       <property name="text">
        <string>手动</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_ave">
       <property name="text">
        <string>平均</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_content">
       <property name="text">
        <string>按内容</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>303</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(218, 218, 218);</string>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <widget class="QPushButton" name="pushButton_new">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>新增</string>
      </property>
     </widget>
     <zorder>pushButton_new</zorder>
     <zorder></zorder>
    </widget>
   </item>
  </layout>
  <zorder>widget</zorder>
  <zorder></zorder>
  <zorder>tabWidget</zorder>
  <zorder>plainTextEdit</zorder>
  <zorder>tabWidget</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>radioButton_manu</zorder>
  <zorder>radioButton_content</zorder>
  <zorder>radioButton_ave</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
